h2o-algos/src/test/java/hex/tree/FriedmanPopescusHTest.java

```diff
@@ -1198,7 +1198,10 @@ public SharedTreeGraph createSharedTreeGraphForTestMLT() {
     
     @Test
     public void testPartialDependenceTree() throws IOException {
-        Frame frame = parseTestFile( "smalldata/h_test_input.csv");
+        String currentPath = new java.io.File(".").getCanonicalPath();
+        Frame frame = parseTestFile(currentPath + "/src/test/resources/hex/tree/h_test_input.csv");
```

Comment:
looks like a local path?

Reply:
yes - I'll remove this and local h_test_input.csv when h_test_input.csv on s3 will be synced on all instances where tests are running - need to write to devops because it's still not synced

Reply:
ok